src/base-commands/twilio-client-command.js

```diff
@@ -40,7 +40,11 @@ class TwilioClientCommand extends BaseCommand {
     this.logger.debug(`Using profile: ${this.currentProfile.id}`);
 
     if (!this.currentProfile.apiKey || !this.currentProfile.apiSecret) {
-      const creds = await this.secureStorage.getCredentials(this.currentProfile.id);
+      // Get Credentials from config file if not there get them from keytar.
```

Comment:
`getProfileById` is called above and I'd expect it to return the profile with the `apiKey` and `apiSecret` already set for non-keytar profiles.

Reply:
`getProfileById` is called above will return the profile with the `apiKey` and `apiSecret` in case of Environment Variables but not for all non-keytar profiles.

Reply:
Updated accordingly.